src/numpy-stubs/__init__.pyi

```diff
@@ -1994,12 +1999,15 @@ class ndarray(_ArrayOrScalarCommon, Generic[_ShapeT_co, _DTypeT_co]):
 
     #
     @property
+    @override
     def real(self: _HasDTypeWithReal[_ScalarT], /) -> ndarray[_ShapeT_co, dtype[_ScalarT]]: ...
     @real.setter
+    @override
```

Comment:
I vaguely remember having seen somewhere that only the getter needs a `@override`... But I can't seem to find anything about that in the typing spec, so I might be wrong here 🤔.